.env.example

```diff
@@ -3,4 +3,5 @@ WOL_URL=
 SHUTDOWN_URL=
 REBOOT_URL=
 LIVENESS_URL=
-COOLDOWN=
\ No newline at end of file
+COOLDOWN=
+POWERBOT_ROLE=
```

Comment:
Only thing I'd request before merge is that we have example values for these params (I'll do the same for the URLs too)

Reply:
Do you want to keep the "default to @everyone" lines, or just fail startup when they're missing, like we do with urls?

Reply:
It won't source from it anyway because it's `.env.example` so I think it's safe to assume if someone creates an `.env` file from it the roles included are on-purpose, so leave the @everyone assumption for default